app/javascript/packs/application.js

```diff
@@ -7,6 +7,8 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
+//= require turbolinks
```

Comment:
do you need these comments?
do you know that lib turbolinks is obsolete now, it was changed to "turbo"

Reply:
@igorkasyanchuk https://github.com/turbolinks/turbolinks
<img width="813" alt="Screenshot 2022-07-03 at 07 25 04" src="https://user-images.githubusercontent.com/31416671/177024502-e8fcd614-e01f-42ff-8901-64b2b82e8d75.png">
Can you please provide some links for it? Cause, official docs shows that we still need to write - turbolinks

Reply:
![image](https://user-images.githubusercontent.com/11101/177054730-6daefcca-0a59-40f1-a4d1-7074711ca10f.png)

and https://guides.rubyonrails.org/working_with_javascript_in_rails.html#turbo-drive

Reply:
@mmrshk ⬆️